nautobot/core/views/generic.py

```diff
@@ -112,6 +115,8 @@ def get(self, request, *args, **kwargs):
             temp_fake_context = {
                 "object": instance,
                 "request": request,
+                "created_by": created_by,
+                "last_updated_by": last_updated_by,
```

Comment:
Doesn't *look* to me like this is actually needed here - this is for getting view configuration, not rendering the view, if I'm reading this right?